pkg/blockatlas/jsonrpc.go

```diff
@@ -16,14 +21,14 @@ type (
 		JsonRpc string      `json:"jsonrpc"`
 		Method  string      `json:"method"`
 		Params  interface{} `json:"params,omitempty"`
-		Id      string      `json:"id,omitempty"`
+		Id      int64       `json:"id,omitempty"`
```

Comment:
Overly strict. As RPC may return string ID, I would not limit to int (still generate numeric ID, but as string)

Reply:
Some upstream rpc doesn’t support string id